robot.py

```diff
@@ -19,6 +30,23 @@ def teleopInit(self):
 
   def teleopPeriodic(self):
     """This function is called periodically during teleoperated mode."""
+    self.strafe = self.pilots.get_strafe_commands()
+    self.turn = self.pilots.get_turn_commands()
+    self.drive = self.pilots.get_drive_commands()
+    self.winch = self.pilots.get_winch_commands()
+    self.hookextend = self.pilots.get_hook_extension_commands()
+    self.hookretract = self.pilots.get_hook_retract_commands()
+    self.hookleft = self.pilots.get_hook_left_commands()
+    self.hookright = self.pilots.get_hook_right_commands()
+
+    self.aim = self.copilots.get_aim_commands()
+    self.fire = self.copilots.get_firing_commands()
+    self.unjam = self.copilots.get_unjam_commands()
+    self.pickup = self.copilots.get_pickup_commands()
+    self.eject = self.copilots.get_eject_commands()
```

Comment:
The teleop function will eventually get very crowded. It will help to create a function to read inputs and call that from this point. That will also allow us to easily collect the operator inputs from other places in the robot.